SoapUI is done

AleksandrDark opened the pull request:
Please could you look at my work?

AleksandrDark committed:
SoapUI is done

GrebNat reviewed (changes requested):
- [x] no setup/teardown scripts
- [ ] for DDT - request is executed N+1 times, where N is number of data. Should be run only N times
- [x] for DDT - test stops execution after first failed iteration
- [x] thanks for different DDT implementations

AleksandrDark committed:
remarks are corrected, except "for DDT - request is executed N+1 times, where N is number of data. Should be run only N times" - This is not a mistake is the feature of the solution.
Test data extract from csv file where
- every even a row are verifiable data
- every odd a row are expectable data.
point 1. 'no setup/teardown scripts' - I added to ChecksCorrectTextTestSuite -> checkTextTestCase -> Setup/TearDown Script

GrebNat reviewed (changes requested):
![image](https://user-images.githubusercontent.com/14941566/137450665-194e5aa1-c5b0-47ad-90a9-91c0d5ad81f5.png)
 
Regarding last comment. № 3 should not appear in the log

AleksandrDark committed:
thank you for your remark, I fixed the remark
